content-repo/extra-docs/articles/CI-CD-Article.md

```diff
@@ -494,6 +494,66 @@ Although you do not have the flexibility of version control and rollback, it is
 </video>
 
 
+## Manage a Pull Request
+
+In order to use this option in Cortex XSOAR platform, download the **XSOAR CI/CD** content pack from Marketplace. (Search for the **Content Management** content pack).
+
+In the pull request management there are 2 options: 
+- Pull request creation.
+- Pull request update.
+
+The platform supports GitHub, GitLab, and Bitbucket.
+The default Git code management is GitHub. If you want to change the default: 
+
+1. In the Cortex XSOAR platform, go to **Playbooks**.
```

Comment:
Maybe instead of all the duplication here and in the next 2 parts - better to write the arguments of the playbook and explanations? 
To merge all these 3 sections of the `playbook Triggered` into one

@ShirleyDenkberg WDYT?

Reply:
@Shellyber How about if we replace those three parts with the following:

To change the default values in the Pull Request Creation - Generic playbook:
1. In the Cortex XSOAR platform, go to **Playbooks**.
2. Search for and select the **Pull Request Creation - Generic** playbook.
3. Click the *Playbook Triggered* task (located in the beginning of the playbook) and update the values as required:
   | Field | Description | Default |
   | --- | --- | ---
   | GitIntegration | Git code management. Possible values are github, gitlab, bitbucket. | github |
   |  MainBranch |  The branch that the pull request will be merged into  | master |
   | ChannelName| The application to send your message to. Possible values are Slack or Teams | ---|
4. Click **Save**.

Reply:
@ShirleyDenkberg  Yes! definitely better